features/api-layer/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -241,10 +241,21 @@
   <reference-list interface="org.opennms.integration.api.v1.ticketing.TicketingPlugin" availability="optional">
     <reference-listener bind-method="onBind" unbind-method="onUnbind" ref="ticketingExtensionManager" />
   </reference-list>
-
   <!-- Service Collectors -->
+  <bean id="rrdConfigStrings" class="java.util.ArrayList">
+    <argument>
+      <list>
+        <value>RRA:AVERAGE:0.5:1:2016</value>
```

Comment:
We need to find a way to make these configuration, I think we can combine this list in a single property and split it on `,`.